packages/documentation-v7/.storybook/main.ts

```diff
@@ -9,17 +9,22 @@ const config: StorybookConfig = {
     name: '@storybook/web-components-vite',
     options: {},
   },
-  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(ts|tsx)'],
+  stories: ['../src/stories/**/*.mdx', '../src/stories/**/*.stories.@(ts|tsx)'],
   addons: [
-    // essential addons
-    '@storybook/addon-essentials',
-
-    // community addons
-    'storybook-dark-mode',
-    // '@pxtrn/storybook-addon-docs-stencil',
     '@storybook/addon-links',
+    '@storybook/addon-a11y',
```

Comment:
We should avoid installing unnecessary modules.
We can add this module, when we really use it.
For now, I would only leave it as a comment (for example).

Reply:
This addon was explicitly added with PR #979 so I think we will need to have it anyway.
I've commented it out for now so we don't forget to add it later.